src/components/Element.ts

```diff
@@ -42,5 +43,11 @@ export function Element(props: ElementProps) {
     setStyle(el, styles);
   }
 
+  if (classOnHover) {
+    const classListOnHover = classOnHover.split(' ');
+    el.onmouseover = () => el.classList.add(...classListOnHover);
```

Comment:
Why the "..." ?

Reply:
to apply the multiple classes, as they are added as this `.add('first', 'second', 'third')` but not as an array of strings